api/src/main/java/io/minio/messages/FilterRule.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Minio Java SDK for Amazon S3 Compatible Cloud Storage, (C) 2017 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.minio.messages;
+
+import org.xmlpull.v1.XmlPullParserException;
+
+import com.google.api.client.util.Key;
+
+
+/**
+ * Helper class to parse Amazon AWS S3 response XML containing filter rule.
+ */
```

Comment:
Instead of AWS S3 response, can we say Minio server response here? It is kind of known that we are S3 compatible, but IMHO we should use Minio in comments etc.